pymatgen/analysis/defects/thermo.py

```diff
@@ -176,22 +191,25 @@ def __post_init__(self):
         entries = []
         for entry in self.phase_diagram.stable_entries:
             d_ = entry.as_dict()
-            d_["energy"] = self.phase_diagram.get_form_energy(entry)
+            d_["energy"] = self.phase_diagram.get_form_energy(entry) - entry.correction
```

Comment:
So this was initially sloppy on my part. Thanks for catching that!
There another place where the correction was subtracted in the PR, but that was in a dictionary without the correction.
I think we can use a more stripped down dict here without the correction as well.